cmd/operator/deploy/operator/03-role.yaml

```diff
@@ -45,34 +45,37 @@ rules:
 - apiGroups: [""]
   resources:
   - configmaps
-  verbs: ["create", "patch", "update"]
+  verbs: ["get", "list", "watch", "create", "patch", "update"]
```

Comment:
are there particular configmaps that you patch and update?

Reply:
Do we know before hand the name of the deployment? If so we can restrict this even more by using resourceNames.

Reply:
I reviewed all gmp-system resource names for each resource type and updated them. Lists and watches have to be under all because you cannot watch multiple resource names in Kubernetes and create must be under all as well because you cannot give create permissions with a name a priori. All other verbs are covered, though.